src/django/pfb_network_connectivity/settings.py

```diff
@@ -271,27 +271,41 @@
+if not PFB_AWS_BATCH_ANALYSIS_JOB_QUEUE_NAME:
+    raise ImproperlyConfigured('env.PFB_AWS_BATCH_ANALYSIS_JOB_QUEUE_NAME is required')
 
 # Configure with either:
-# 1. PFB_AWS_BATCH_JOB_DEFINITION_NAME_REVISION = '<name>:<revision>'
+# 1. PFB_AWS_BATCH_ANALYSIS_JOB_DEFINITION_NAME_REVISION = '<name>:<revision>'
 #  -- or --
-# 2. PFB_AWS_BATCH_JOB_DEFINITION_NAME = '<name>'
+# 2. PFB_AWS_BATCH_ANALYSIS_JOB_DEFINITION_NAME = '<name>'
 #   In this case, revision will be autodetected by querying the AWS API
 # Case 1 takes precedence and is the value set for use in the app
-PFB_AWS_BATCH_JOB_DEFINITION_NAME_REVISION = os.getenv('PFB_AWS_BATCH_JOB_DEFINITION_NAME_REVISION')
-PFB_AWS_BATCH_JOB_DEFINITION_NAME = os.getenv('PFB_AWS_BATCH_JOB_DEFINITION_NAME')
-if PFB_AWS_BATCH_JOB_DEFINITION_NAME_REVISION:
+PFB_AWS_BATCH_ANALYSIS_JOB_DEFINITION_NAME_REVISION = os.getenv('PFB_AWS_BATCH_ANALYSIS_JOB_DEFINITION_NAME_REVISION')
+PFB_AWS_BATCH_ANALYSIS_JOB_DEFINITION_NAME = os.getenv('PFB_AWS_BATCH_ANALYSIS_JOB_DEFINITION_NAME')
+if PFB_AWS_BATCH_ANALYSIS_JOB_DEFINITION_NAME_REVISION:
     pass
-elif PFB_AWS_BATCH_JOB_DEFINITION_NAME:
-    revision = get_latest_job_definition(PFB_AWS_BATCH_JOB_DEFINITION_NAME)['revision']
-    PFB_AWS_BATCH_JOB_DEFINITION_NAME_REVISION = '{}:{}'.format(PFB_AWS_BATCH_JOB_DEFINITION_NAME,
-                                                                revision)
+elif PFB_AWS_BATCH_ANALYSIS_JOB_DEFINITION_NAME:
+    revision = get_latest_job_definition(PFB_AWS_BATCH_ANALYSIS_JOB_DEFINITION_NAME)['revision']
+    PFB_AWS_BATCH_ANALYSIS_JOB_DEFINITION_NAME_REVISION = '{}:{}'.format(PFB_AWS_BATCH_ANALYSIS_JOB_DEFINITION_NAME,
+                                                                         revision)
 else:
-    raise ImproperlyConfigured('env.PFB_AWS_BATCH_JOB_DEFINITION_NAME_REVISION or ' +
-                               'env.PFB_AWS_BATCH_JOB_DEFINITION_NAME is required.')
+    raise ImproperlyConfigured('env.PFB_AWS_BATCH_ANALYSIS_JOB_DEFINITION_NAME_REVISION or ' +
+                               'env.PFB_AWS_BATCH_ANALYSIS_JOB_DEFINITION_NAME is required.')
+
+
+# Same setup for tilemaker jobs, but without the exception-throwing
+PFB_AWS_BATCH_TILEMAKER_JOB_QUEUE_NAME = os.getenv('PFB_AWS_BATCH_TILEMAKER_JOB_QUEUE_NAME')
+PFB_AWS_BATCH_TILEMAKER_JOB_DEFINITION_NAME_REVISION = os.getenv('PFB_AWS_BATCH_TILEMAKER_JOB_DEFINITION_NAME_REVISION')
+PFB_AWS_BATCH_TILEMAKER_JOB_DEFINITION_NAME = os.getenv('PFB_AWS_BATCH_TILEMAKER_JOB_DEFINITION_NAME')
+if PFB_AWS_BATCH_TILEMAKER_JOB_DEFINITION_NAME and not PFB_AWS_BATCH_TILEMAKER_JOB_DEFINITION_NAME_REVISION:
+    try:
+        revision = get_latest_job_definition(PFB_AWS_BATCH_TILEMAKER_JOB_DEFINITION_NAME)['revision']
+    except NoActiveJobDefinitionRevision:
+        pass
+    PFB_AWS_BATCH_TILEMAKER_JOB_DEFINITION_NAME_REVISION = '{}:{}'.format(PFB_AWS_BATCH_TILEMAKER_JOB_DEFINITION_NAME,
```

Comment:
Yeah, you're right.  No reason for this to act any differently from the analysis job setup.  I changed it to match.